vpc/tool/gc3/gc.go

```diff
@@ -52,8 +50,6 @@ func GC(ctx context.Context, timeout time.Duration, instanceIdentityProvider ide
 	switch args.SourceOfTruth {
```

Comment:
What do you think about removing switch statements like this? Second pass maybe, or keep in place for future nomad support?

Reply:
I was thinking for leaving it when we add openstack support.

[Kidding, will remove in a second pass]